MenuItem.php

```diff
@@ -13,6 +13,8 @@ class MenuItem extends AbstractItem {
     /** @var string icon file */
     protected $svg = __DIR__ . '/images/rename.svg';
 
+    protected $type = "plugin_move_page";
```

Comment:
This completely breaks displaying the non-mobile rename menu item:

![Screenshot 2022-01-24-move-link](https://user-images.githubusercontent.com/198317/150867138-ff31f924-a5f6-4ba3-b76c-23ac2123bc84.png)

I think this is because the plugin still contains some CSS for the menu item produced by `TEMPLATE_PAGETOOLS_DISPLAY` that is still handled in the plugin and due to this change, this CSS is now applied to the new menu item. I don't know if there are still templates for which this event and the corresponding CSS is relevant.

Reply:
ah, yes these old styles clashed. The handling for that event should be removed, but for now I simply renamed the class.